app/views/pages/sponsor.html.haml

```diff
@@ -0,0 +1,107 @@
+          %ul
+            %li 35% women (double the national average of 18%)
+            %li 18% minority ethnicities in technology (three times the national average of 5%)
+
+      .large-6.columns.show-for-large-up 
+        #sponsor-map-jumbotron
+
+    %h4 What our hackers have been up to
+    %p
+      :markdown
+        At the top two collegiate hackathons in the world, [PennApps
+        X](http://2014f.pennapps.com/) and [MHacks
+        V](http://mhacksv.challengepost.com/), half of the six winning teams
+        were led by high school students. All of these winning high school teams
+        are part of hackEDU.
+
+        Most recently, [Rohith Varanasi](http://rohithvaranasi.com/), one of
+        our club leaders from South Brunswick High School, and the rest of his
+        team from [MHacks IV](http://mhacks-iv.challengepost.com/) were named
+        the [Major League Hacking + Thiel 2014 Rookie of the
+        Year](http://news.mlh.io/cosmos-thiel-rookie-year-2014-mlh-fall-season-02-09-2015)
+        for creating [Cosmos
+        Browser](http://www.androidauthority.com/cosmos-browser-sms-data-connection-526654/).
+        Another one of our leaders, [Raphael Kats](http://raphaelrk.com/),
+        created [DataWave](http://challengepost.com/software/datawave), another
+        data-less browser using radio waves at MHacks V and won 2nd place.
+
+.row
+  %h2.sponsorship.text-center Sponsorship Tiers
+  .small-12.medium-4.columns
+    %ul.pricing-table
+      %li.title Triassic
+      %li.price $3,000
+      %li.bullet-item Logo on website/marketing materials
+      %li.bullet-item.invalid-item Blast emails/Facebook messages to club leaders
+      %li.bullet-item.invalid-item Facilitated introductions to top hackers
+      %li.bullet-item.invalid-item Branded prize
+      %li.bullet-item.invalid-item Blurb on website
+      %li.bullet-item Ship swag to clubs
+      %li.bullet-item.invalid-item Opt-in email list
+  .small-12.medium-4.columns
+    %ul.pricing-table
+      %li.title Jurassic
+      %li.price $7,500
+      %li.bullet-item Logo on website/marketing materials
+      %li.bullet-item Blast two emails/Facebook messages to club leaders
+      %li.bullet-item Eight facilitated introductions to top hackers
+      %li.bullet-item.invalid-item Branded prize
+      %li.bullet-item.invalid-item Blurb on website
+      %li.bullet-item Ship swag to clubs
+      %li.bullet-item.invalid-item Opt-in email list
+  .small-12.medium-4.columns
+    %ul.pricing-table
+      %li.title Cretaceous
+      %li.price $12,000
+      %li.bullet-item Logo on website/marketing materials
+      %li.bullet-item Blast four emails/Facebook messages to club leaders
+      %li.bullet-item Sixteen facilitated introductions to top hackers
+      %li.bullet-item Branded prize
+      %li.bullet-item Blurb on website
+      %li.bullet-item Ship swag to clubs
+      %li.bullet-item Opt-in email list
+
+%script
+  markersJSON = #{raw@clubs_hash.to_json}
```

Comment:
There should be a space between `raw` and `@clubs_hash.to_json`. See `pages/home.html.haml` for an example.